_posts/2016-07-01-newsletter.md

```diff
@@ -0,0 +1,254 @@
+  `nvim`-based backend for his well-known
+  **[VimR](https://github.com/qvacua/vimr) project**, a polished GUI frontend for macOS.
+- [neovim-qt](https://github.com/equalsraf/neovim-qt) continues to get better
+  and better: **performance now matches gVim**, with less flicker when scrolling.
+- [nvim-hs](https://github.com/neovimhaskell/nvim-hs) is a Haskell host for
+  Neovim plugins. This means you can **write Neovim plugins in Haskell**.
+- Keep an eye on [cl-neovim](https://github.com/adolenc/cl-neovim), which is
+  seeing consistent updates. It enables **lisp plugins** and fun things
+  like `:Lispdo`.
+- [nvr](https://github.com/mhinz/neovim-remote) recently published a release
+  that provides most or all of the legacy Vim "clientserver"-related
+  command-line options, such as `--servername`, `--remote`, etc. `nvr` is
+  perfect for communicating with a parent `nvim` instance from inside
+  a `:terminal` buffer.
+- Intero users are excited about these new projects:
+  [intero.nvim](https://github.com/myfreeweb/intero.nvim) and
+  [intero-neovim](https://github.com/parsonsmatt/intero-neovim)
+- @carlosdcastillo continues to update the Neovim-based
+  [**Atom** nvim-mode](https://github.com/carlosdcastillo/vim-mode).
+  Have you tried it?
+
+There are clients for go, julia, perl, Java, R, Elixir, and Clojure. You really
+should go the [related projects](https://github.com/neovim/neovim/wiki/Related-projects)
+wiki page whenever you're curious about new work.
+
+And some long-awaited features have landed in `nvim` core.
+
+- ruby support is now in master, release 0.1.5. This means you can write Neovim
+  plugins in ruby, _and_ the legacy Vim `:ruby`, `:rubydo`, and `:rubyfile`
+  commands are supported (so existing Vim+ruby plugins work in Neovim, such as
+  Command-t and vim-github-dashboard).
+  - To enable Ruby support, just `gem install neovim`. You don't need to worry
+    about compiling against a specific version.
+- `:CheckHealth` is like homebrew "doctor", conceived by @tweekmonster and
+  extended by @tjdevries. We look forward to adding more and more "sanity
+  checks" covering common problems. In the future, instead of wondering whether
+  your runtime is broken or your `$TERM` is wrong (or ...), reach for
+  `:CheckHealth`. Users should not need to fumble with the problems that
+  experienced users just "learn to deal with".
+
+Funding: Nikolai Pavlov (ZyX) assumes Thiago's role
+-------
+
+The enormously successful
+[salt campaign](https://salt.bountysource.com/teams/neovim) has funded
+Thiago's work, yielding libmpack, major refactors and improvements. For personal
+reasons Thiago will take a less active role in the project, and the 2-3 days of
+funded work will now be fulfilled by ZyX, the second-most [prolific](#note1)
+committer to Neovim _and Vim_ (~50,000 LOC contributed to Neovim, ~13,000 LOC
+contributed to Vim—including `if_pyth`, Vim's most mature FFI).
```

Comment:
maybe `if_pyth` is technically correct in some sense (the help file is called that at least), but doesn't `if_python` sound better?